assets/style/about/about.scss

```
@@ -6,9 +6,536 @@
+          }
+          @media (max-width: $phone-l) {
+            max-width: 100%;
+            overflow: scroll;
+            height: 400px;
+            margin-bottom: 50px;
+          }
+        }
+      }
+
+      .main__history {
+        min-height: 340px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        .main__wr-history-year {
+          display: flex;
+          justify-content: space-between;
+          @media (max-width: $laptop-m) {
+            flex-wrap: wrap;
+            align-items: center;
+            min-height: 500px;
+          }
+
+          .main__history-year {
+            display: flex;
+            flex-direction: column;
+            align-items: flex-start;
+            font-family: $font;
+            min-height: 200px;
+            justify-content: space-between;
+            @media (max-width: $pc-s) {
+              text-align: center;
+              align-self: center;
+            }
+            @media (max-width: $laptop-m) {
+              flex-basis: 32%;
+              align-items: center;
+            }
+            @media (max-width: $phone-l) {
+              flex-basis: 49%;
+              min-height: 210px;
+              margin-bottom: 10px;
+            }
+
+            .main__year {
+              padding: 23px 15px 25px 15px;
+              font-weight: 500;
+              font-size: 20px;
+              line-height: 28px;
+              color: $text-light-color;
+              border-radius: 50%;
+              @media (max-width: $pc-s) {
+                text-align: center;
+                align-self: center;
+              }
+              @media (max-width: $phone-l) {
+                font-size: 18px;
+                padding: 20px 13px 18px 13px;
+                margin-bottom: 5px;
+              }
+            }
+            .main__year_1 {
+              background: $btn-link-dark-color;
+            }
+            .main__year_2 {
+              background: #30569c;
+            }
+            .main__year_3 {
+              background: #225dca;
+            }
+            .main__year_4 {
+              background: #3972dc;
+            }
+            .main__year_5 {
+              background: #236cf2;
+            }
+            .main__year_6 {
+              background: #2568e3;
+            }
+
+            .main__year-descr {
+              max-width: 215px;
+              font-weight: 400;
+              font-size: 16px;
+              line-height: 18px;
+              color: $text-blue-color;
+              @media (max-width: $tablet-s) {
+                font-size: 14px;
+              }
+              @media (max-width: $tablet-s) {
+                font-size: 13px;
+              }
+              @media (max-width: $phone-l) {
+                max-width: 90%;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  &__wr-employees-partners {
+    background-color: #e5e5e5;
```

Здесь вроде бы другой цвет по фигме
```suggestion
    background-color: #f1f4f8;
```